Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -482,7 +486,10 @@ namespace AZ::SceneAPI::Behaviors
         // The originPath we pass to LoadTemplateFromString must be the relative path of the file
         AZ::IO::Path templateName(prefabGroup->GetName());
         templateName.ReplaceExtension(AZ::Prefab::PrefabGroupAssetHandler::s_Extension);
-        templateName = relativePath / templateName;
+        if (!AZ::StringFunc::StartsWith(templateName.c_str(), relativePath.c_str()))
```

Comment:
I'd be careful with string comparison here with relativePath. Are we guaranteed that relativePath always has the '/' as the path separator?

Reply:
it is really an asset path, not a "native path" which all use the forward slash